README.md

```diff
@@ -1411,6 +1411,9 @@ All notable changes to this project will be documented in this file.
 - [relnotes](https://crates.io/crates/relnotes) - A tool to automatically generate release notes for your project.
 - [cocogitto](https://github.com/oknozor/cocogitto) - A set of CLI tools for the conventional commit
 and semver specifications.
+- [@favware/cliff-jumper](https://github.com/favware/cliff-jumper) - A NodeJS CLI tool that combines git-cliff and
```

Comment:
Can you list it as `cliff-jumper` instead of `@favware/cliff-jumper`?

Reply:
```suggestion
- [cliff-jumper](https://github.com/favware/cliff-jumper) - A NodeJS CLI tool that combines git-cliff and
```

just gonna do that this way so I don't have to switch to fork :^)